js_modules/dagit/packages/core/src/assets/AssetLineageElements.tsx

```diff
@@ -46,17 +44,7 @@ const AssetLineageInfoElement: React.FC<{
           <Box flex={{display: 'inline-flex', alignItems: 'center'}}>
             {lineage_info.assetKey.path
               .map<React.ReactNode>((p, i) => <span key={i}>{p}</span>)
-              .reduce((prev, curr, i) => [
-                prev,
-                <Box key={`separator_${i}`} padding={{horizontal: 2}}>
-                  <Icon
-                    icon={IconNames.CHEVRON_RIGHT}
-                    iconSize={11}
-                    style={{position: 'relative', top: '-1px'}}
-                  />
-                </Box>,
-                curr,
-              ])}
+              .reduce((prev, curr) => [prev, <>&nbsp;{'>'}&nbsp;</>, curr])}
```

Comment:
I /think/ these React fragments probably need to have keys? I'm not 100% sure though?